backend/api/v1/auth_service.go

```diff
@@ -121,12 +121,9 @@ func (s *AuthService) CreateUser(ctx context.Context, request *v1pb.CreateUserRe
 		}
 	}
 
-	// Try to find out if the email is used by Bytebase user.
-	emptyIdentityProviderResourceID := ""
 	existingUser, err := s.store.GetUser(ctx, &store.FindUserMessage{
-		Email:                      &request.User.Email,
-		ShowDeleted:                true,
-		IdentityProviderResourceID: &emptyIdentityProviderResourceID,
+		Email:       &request.User.Email,
```

Comment:
We need to construct the email when the username is primary.

Reply:
Maybe not, it's sign in with email and password in bytebase console, not SSO.